packages/documentation/src/components/dashboard/DashboardDataFetch.jsx

```diff
@@ -142,22 +149,28 @@ export async function deploysFetch(
 
     // If commit wasn't deployed by a full build, check for single-app build
     let hasSuccessfulSingleAppBuild = false;
+    let isContinuousDeploymentEnabled = false;
     if (isVetsWebsite && (!isOnDev || !isOnStaging || !isOnProd)) {
       const buildArtifactUrl = `${repo.buildArtifacts}/${sha}.txt`;
       const response = await fetch(buildArtifactUrl);
       if (response.ok) {
         const fileText = await response.text();
-        const matchText = fileText.match(/IS_SINGLE_APP_BUILD=(\w+)/)[1];
-        const isSingleAppBuild = matchText === 'true';
+        const isSingleAppBuild =
+          getValueFromFile(fileText, 'IS_SINGLE_APP_BUILD') === 'true';
```

Comment:
At some point we can create constants for these values, but it probably isn't needed at this point

Reply:
Good point, if this expands in the future that would be useful to add